experimenter/experimenter/features/tests/__init__.py

```diff
@@ -1,37 +1,25 @@
-import os
-import pathlib
+from pathlib import Path
 
 from django.test import override_settings
 
+FIXTURE_DIR = Path(__file__).parent.absolute() / "fixtures"
+
 mock_valid_features = override_settings(
-    FEATURE_MANIFESTS_PATH=os.path.join(
-        pathlib.Path(__file__).parent.absolute(), "fixtures", "valid_features"
-    )
+    FEATURE_MANIFESTS_PATH=FIXTURE_DIR / "valid_features"
```

Comment:
Can we update these to be using the `Path(x, x, x, ...)` syntax?

Reply:
I think if we wanted to set some consistency around that we'd want a lint rule, otherwise there's no good way to enforce it.  Maybe ruff will add rules around it, but barring that we're already using this syntax in other places so it's probably not worth fretting about it here.

Reply:
Using `/` is idiomatic when the left-hand side is a pth.

If `FIXTURE_DIR` were a string, then we would have to do `Path(FIXTURE_DIR, "valid_features")`